pkg/controller/download_request_controller.go

```diff
@@ -169,10 +169,10 @@ func (r *downloadRequestReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 			Name:      backup.Spec.StorageLocation,
 		}, location); err != nil {
 			if apierrors.IsNotFound(err) {
-				log.Errorf("BSL for DownloadRequest cannot be found")
+				log.Errorf("fail to get BSL for DownloadRequest")
```

Comment:
here isn't better to have a different message from below how it was?

Reply:
mistake, thanks review.